### PR TITLE
Programme and dates - WIP

### PR DESCRIPTION
**Pull request by evildmp**

### CODE REVIEW

File: source/dates.rst [hunk 1]
```diff
@@ -1,19 +1,36 @@
...
 
-2015
-----
+    * 14 - General programme announced
 
-* Saturday 14th November - Wash-up Meeting in Cardiff
+  * May
 
-  - :doc:`wash-up-meeting-report`
+    * 10 Call for proposals closes
```
Comment (Lukasa) [inline source/dates.rst]: Missing a dash here

File: source/dates.rst [hunk 2]
```diff
@@ -1,19 +1,36 @@
...
+    * 15
+
+      * Ticket sales & CFP open for under-represented groups
+      * financial assistance applications open
+
+* 2015
+
+  * November
+
+    * 14 :doc:`Wash-up Meeting in Cardiff <wash-up-meeting-report>`
```
Comment (Lukasa) [inline source/dates.rst]: Missing a dash here too.

File: source/programme-talks.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
+
+About 57 of the speakers were male, and the vast majority were of course
+extremely white.
```
Comment (Lukasa) [inline source/programme-talks.rst]: I agree with the sentiment, but I feel like "extremely" is not the world to use here. Suggests a tone I don't think we want.

File: source/programme-talks.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Some of our best friends are white men, and there is nothing wrong with
+us^wthem, but all the same some of best friends are not, and for some reason the
```
Comment (Lukasa) [inline source/programme-talks.rst]: "some of _our_ best friends"

File: source/programme-talks.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+ones who aren't didn't come forward to speak at PyCon.
```
Comment (Lukasa) [inline source/programme-talks.rst]: This is a real bike shed, but: do we only want under-represented groups because they're our best friends? I feel like that's limiting: maybe consider changing "our best friends" to "our community members" or something similar?

**Comment by Lukasa** [comment]
Some small style notes here, but I really like the shape of this!

File: source/programme-talks.rst (hunk 3 above)
Comment (evildmp) [inline source/programme-talks.rst]: None of this is intended for public consumption, nor am I suggesting that it should make its way into the published site!

File: source/programme-talks.rst (hunk 5 above)
Comment (evildmp) [inline source/programme-talks.rst]: It's not really about best friends at all...

File: source/dates.rst [hunk 6]
```diff
@@ -1,19 +1,36 @@
...
+
+  * April
+
+    * 12 - Ticket sales & CFP open for everyone
+
+  * March
+
+    * 15
+
+      * Ticket sales & CFP open for under-represented groups
```
Comment (inglesp) [inline source/dates.rst]: We talked about doing this last year, but decided against it.  ("I'm not sure how much difference this will make. To me it seems a little condescending, I guess. Also I think that people in underrepresented groups have as much trouble deciding what they'll be doing several months in advance as anyone else, maybe more so.")

File: source/programme-talks.rst [hunk 7]
```diff
@@ -0,0 +1,74 @@
+Programme of talks
+==================
+
+At PyCon UK 2015, we had approximately 70 talks.
```
Comment (inglesp) [inline source/programme-talks.rst]: Could we have a separate page in the docs about diversity of speaker line-up, linked to from this page?

File: source/programme-talks.rst (hunk 3 above)
Comment (inglesp) [inline source/programme-talks.rst]: Note that this repo _is_ public.

File: source/programme-talks.rst [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+
+About 57 of the speakers were male, and the vast majority were of course
+extremely white.
+
+Some of our best friends are white men, and there is nothing wrong with
+us^wthem, but all the same some of our best friends are not, and for some reason
+the ones who aren't didn't come forward to speak at PyCon.
```
Comment (inglesp) [inline source/programme-talks.rst]: I know what you're saying here, but find "aren't didn't" extremely hard to parse.  Also, "us^wthem" suggests that the conference committee is all white me, which it's not.

For this kind of writing, I think it's important to be as clear as possible.

What's wrong with saying "We are concerned that the conference line-up has been too white and too male, and we plan to work hard to ensure this is not the case in 2016."?

File: source/programme-talks.rst [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+
+The simple solution to this is to get more of our friends who are not white men
+to propose talks.
+
+Target
+------
+
+I will be disappointed if at this PyCon we don't receive at least **one-third of
```
Comment (inglesp) [inline source/programme-talks.rst]: It's not clear who "I" is here.  Since this document is going to be part of our conference policy, can you change "I" to "we", perhaps changing "I will" to "we should"?

File: source/programme-talks.rst [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+This is not impossible to achieve, by any means.
+
+PyCon US:
+
+* 2011: about 1% of speakers were women
+* 2014 and 2015: about one-third were women
+
+DjangoCon Europe:
+
+* 2014: two our of 32 speakers were women
```
Comment (inglesp) [inline source/programme-talks.rst]: s/our/out/

File: source/programme-talks.rst [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+* 2105: approximately one-third of speakers were women
+* 2016: approximately one-half of speakers are women
+
+It's less easy to put numbers to other under-represented groups.
+
+
+How I want to achieve this
```
Comment (inglesp) [inline source/programme-talks.rst]: s/How I want to/How we can/

**Comment by inglesp** [comment]
Hi @evildmp, thanks for this.  I agree with the sentiment completely, but have made some style/tone suggestions.  They're just suggestions though!

**Comment by Lukasa** [comment]
Just kicking Travis.

**Comment by inglesp** [comment]
Ping @evildmp -- what're your thoughts on the feedback?

**Comment by inglesp** [comment]
Hi @evildmp -- thanks for the updates, this is looking good.  There's some duplication between the speaker-diversity page and the programme-talks page -- could you fix?

**Comment by evildmp** [comment]
@inglesp the old `programme.rst` page should have been removed (it is now).

File: source/speaker-diversity.rst [hunk 12]
```diff
@@ -0,0 +1,69 @@
+Speaker diversity
+=====================================
+
+PyCon UK 2016 aims to present a high-quality and balanced programme of talks and other sessions
+(workshops, sprints, panels).
+
+These sessions will represent the diversity of interests within the Python community, and also the
```
Comment (doismellburning) [inline source/speaker-diversity.rst:7]: > will

I appreciate the bullishness, but given how easy it is to fuck up, perhaps "aim to"?
